### DISCUSSION TITLE
Using GitHub for personal content projects

### DISCUSSION THREAD

**Discussion by varunsridharan** [community/community#21728] [New to GitHub]
<p>Hi There</p>
<p>Is there something else for Github Personal Account ( <a href="https://github.blog/changelog/2019-02-21-organization-wide-community-health-files/" rel="nofollow noopener">https://github.blog/changelog/2019-02-21-organization-wide-community-health-files/</a> )</p>
<p>I mean like i work on quite a lot of projects. and i have a basic template for my projects on how i need the repo to be.</p>
<p>for eg</p>
<pre><code>.gitignore
.gitattributes
README.md
</code></pre>
<p>And so on.</p>
<p>it would be awesome if i can create a repo where i can add all this and these files will be used automatically when i create a new repo.</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/varunsridharan">@varunsridharan</a>,</p>
<p>Currently this is only for organizations, though you can <a href="https://help.github.com/en/articles/creating-a-new-organization-from-scratch" rel="nofollow noopener">create an organization for your personal projects</a>. I’ve passed on your suggestion to our internal team, however, and if this gets released for personal accounts, it will be announced in the <a href="https://github.blog/changelog/" rel="nofollow noopener">GitHub Changelog</a>.</p>
<p>Thanks!</p>

**Comment by varunsridharan** [comment]
<p>Finally Got An Answer :smiley: OMG&gt;</p>
<p>thanks for passing this to your team. but please it would be a really great feature.</p>

**Comment by varunsridharan** [comment]
<p>Hi There</p>
<p>I am not asking about Community Files for  organization . i am asking for a way how i can have a github repo template copied to new repo i create.</p>
<p>for eg here is my common github basic template <a href="http://github.com/varunsridharan/git-template/" rel="nofollow noopener">http://github.com/varunsridharan/git-template/</a></p>
<p>in which i have quite a few files and i need those when ever i create a brand new repo. in github and the same applies if i do it in a organization</p>

**Comment by ThirdEyeRose** [comment]
<p>Thanks for clarifying, <a class="mention" href="/varunsridharan">@varunsridharan</a>. This is currently not possible, but it is also something that our product team is investigating. In the meantime, the best solution is to do as you’ve done and create a template repository that you can copy over when you create a new repository.</p>
<p>Thanks!</p>
